src/main/scala/edu/ie3/simona/model/participant/BmModel.scala

```diff
@@ -105,8 +117,8 @@ final case class BmModel(
       usage: Double,
       eff: Double,
   ): Power = {
-    val currOpex = opex / eff
-    val avgOpex = (currOpex + opex) / 2
+    val currCapex = capex / eff
+    val avgOpex = (currCapex + capex) / 2
```

Comment:
Why did you change this? The `opex` is the value, we want here. Could you change this back?